src/components/forms/SelectRoleForm.js

```diff
@@ -252,8 +251,7 @@ export default enhance(
             </Column>
             {touched.roles && errors.roles && <div>{errors.roles}</div>}
           </Row>
-          {/* TODO: Issue #321 */}
-          {/* <Row buffer>
+          <Row buffer>
```

Comment:
should we nix `buffer` now that we can use `<Row mt={2}>` ? it's only a 2px difference, and means we're not using custom helpers that don't respect the theme spacing

Reply:
YES! will add this.